### DISCUSSION TITLE
BroadLink codici IR condizionatore

### DISCUSSION THREAD

**Discussion by Heisenb3rg84** [indomus/forum#404] [HUB: Home Assistant]
Buonasera a tutti,
 premetto che scrivo dopo aver letto diversi articoli sul sito e fatto diverse prove sul mio sistema.
Sto cercando di memorizzare i codici IR del telecomando del climatizzatore (un Fujitsu vecchio come me probabilmente) e mi sono imbattuto in un comportamento che non mi aspettavo.
La registrazione del codice la faccio senza modificare le impostazioni (temperatura, velocità della ventola, modalità, etc..) ma premendo solo il tasto di accensione.
A registrazione avvenuta provvedo a spegnere il telecomando avendo cura di non premere altri tasti.
Ripeto la registrazione premendo nuovamente solo il tasto di accensione (quindi con la medesima configurazione del tentativo precedente).
Come risultato non ottengo lo stesso codice ma due distinti, cosa che non mi aspettavo.
Avevo inteso che una determinata combinazione (intesa come modalità, velocità ventola, temperatura, etc...) corrispondesse a un codice ben preciso.
Ho inteso male?
Se si come si capisce qual'è il codice "buono"?
Vi ringrazio per il tempo che mi dedicherete.

**Comment by KC-inDomus** [comment]
questa cosa capitare capita. il tasto di accensione è un tasto un po' particolare, specie in certi modelli datati.
la cosa che farei (e faccio) io è quella di accendere il telecomando e da lì, col telecomando già acceso, derivare i codici muovendomi da tra un'impostazione e l'altra (ovviamente tutto dipende poi dalle funzioni che ha il telecomando in sé)

**Reply by Heisenb3rg84** [reply]
Ciao,
 grazie per la risposta.
Anche a telecomando acceso il discorso non cambia.
Ad esempio mantenendo la modalità Dry, con fan in Auto procedo a modificare solo la temperatura (di 1 grado).
Quelli che seguono sono i 4 risultati che ho ottenuto: abc_1 e abc_2 sono la stessa configurazione: a telecomando acceso ho catturato i 22°, dopo la registrazione mi sono riportato a 21°, l'ho fatta partire di nuovo e l'ho riportato a 22°.
abc_3 e abc_4 idem come sopra, cambia solo la temperatura di riferimento (che è inferiore di 1 grado).
A quanto pare sembra non essere un problema del tasto di avvio (come ipotizzato da te inizialmente).


```
"abc_1": "JgD2AGY1Dg0ODQ0pDQ0OKA0ODQ4NDQ4oDSkNDQ4NDg0NKQ0oDQ4NDg0ODQ0ODQ0ODQ4NDg0NDg0NDg0ODQ4NKA4NDQ4NDg0NDg0NDg0ODSgODQ0ODQ4NDQ4NDigNKA4oDSkNKA4oDQ4NDg0NDSkNDg0NDg0NDg0ODQ4NDQ4NDSkNKA4NDQ4NDg0ODQ0ODQ0pDQ0OKA0ODQ4NKA4NDQ4NDg0NDg0ODQ0pDSgODQ0ODQ4NDQ4NDQ4NDg0ODQ0ODQ0ODQ4NDQ4NDg0NDg0ODQ0ODQ4NDQ4NDg0NDg0ODQ0ODQ4NDQ4NDg0NKQ0oDg0NKQ0oDSkNKA4NDQANBQ==",
"abc_2": "JgD2AGY1Dg0ODQ0pDQ0OKA0ODQ4NDQ4oDSkNDQ4NDg0NKQ0oDg0NDg0ODQ0ODQ0ODQ4NDg0NDg0NDg0ODQ4NKA4NDQ4NDg0NDg0NDg0ODSgODQ0ODQ4NDQ4NDigNKA4oDSgOKA0pDQ4NDQ4NDSkNDg0NDg0NDg0ODQ0ODQ4NDSkNKA4NDQ4NDg0ODQ0ODQ0pDQ0OKA0ODQ4NKA0ODQ4NDg0NDg0ODQ0oDigNDg0ODQ0ODQ4NDQ4NDg0NDg0ODQ0ODQ4NDQ4NDg0NDg0ODQ0ODQ4NDQ4NDg0NDg0ODQ0ODQ4NDQ4NDQ4NKA4oDQ4NKQ0oDSkNKA4NDQANBQ==",
"abc_3": "JgD2AGY2DQ0ODQ4oDQ4NKA4NDQ4NDg0oDigNDg0NDg0NKQ0oDg0NDg0ODQ4NDQ4NDQ4NDg0NDg0ODQ0ODQ4NKA4NDQ4NDg0NDg0ODQ0ODSkNDQ4NDQ4NDg0NDigNKQ0oDSkNKA4oDQ4NDQ4NDigNDg0NDg0ODQ0ODQ4NDQ4NDigNKA4NDg0NDg0ODQ4NDQ0ODSkNKA0ODQ4NKA4NDQ4NDg0NDg0ODQ0pDSgODQ0ODQ4NDQ4NDg0NDg0ODQ0ODQ4NDQ4NDg0NDg0ODQ0ODQ4NDQ4NDQ4NDg0ODQ0ODQ4NDQ4NDg0NDg0NKQ0oDwwNKQ0NDigNKQ0NDgANBQ==",
"abc_4": "JgD2AGY2DQ0ODQ0pDQ0OKA0ODQ4NDg0oDSkNDQ4NDg0NKQ0oDg0NDg0ODQ0ODQ0ODQ4NDg0NDg0NDg0ODQ4NKA0ODQ4NDg0NDg0NDg0ODSgODQ0ODQ4NDQ4NDigNKA4oDSkNKA4oDQ4NDQ4NDSkNDg0NDg0NDg0ODQ4NDQ4NDSkNKA4NDQ4NDg0ODQ0ODQ0ODSgOKA0ODQ4NKA0ODQ4NDg0NDg0ODQ0oDigNDg0ODQ0ODQ4NDQ4NDg0NDg0ODQ0ODQ4NDg0NDg0NDg0ODQ0ODQ0ODQ4NDg0NDg0ODQ0ODQ4NDQ4NDQ4NKQ0oDQ4NKQ0NDigNKA4NDgANBQ=="```

**Reply by KC-inDomus** [reply]
se sei sicuro al 100% che i codici siano gli stessi, quelli devi usare, perché quelli - usando il telecomando - il condizionatore riceve.

**Reply by Heisenb3rg84** [reply]
I 4 codici che ho riportato sopra avrebbero dovuto essere uguali (2 a 2) invece sono tutti e 4 diversi.
Tu hai mai provato a catturare il codice della stessa configurazione più volte? E' sempre uguale?

**Reply by KC-inDomus** [reply]
a volta variano. il punto è che non è mai perfettamente identico, spesse volte per questioni di intensità del segnale, precisione nella lettura ecc. sono corretti a valle, lato condizionatore.
